buildenv/jenkins/common/variables-functions

```diff
@@ -675,12 +675,19 @@ def set_build_identifier() {
     if (!BUILD_IDENTIFIER) {
         BUILD_IDENTIFIER = ''
         if (params.ghprbPullId && params.ghprbGhRepository) {
+            // If Pull Request build
             BUILD_IDENTIFIER = params.ghprbGhRepository + '#' + params.ghprbPullId
+        } else if (PERSONAL_BUILD) {
+            // If Personal build
+            wrap([$class: 'BuildUser']) {
+                BUILD_IDENTIFIER = "${BUILD_USER_EMAIL}"
+            }
+        } else {
+            echo "WARNING: Can't determine appropriate BUILD_DENTIFIER automatically. Please add it as a build parameter if you would like it set."
         }
     }
     if (BUILD_IDENTIFIER) {
-        def TMP_DESC = (currentBuild.description) ? currentBuild.description + "<br>" : ""
-        currentBuild.description = TMP_DESC + BUILD_IDENTIFIER
```

Comment:
Is the description set somewhere else? I don't see it.

Reply:
No, the `BUILD_IDENTIFIER ` is moved to the `displayName` instead of the `description`.

Reply:
I see the following code appending to description, is it going to be a problem if description  was never set?
```
urrentBuild.description += "<br/>${params.PLATFORMS}"
```

Reply:
I assume you're referring to
https://github.com/eclipse/openj9/pull/3863/files#diff-561308a6a13d3e98f6c52cacbf3b4ddaL124
This should still be fine since the SHAs are added to the desc before the platforms.
Ex:
```
OpenJ9: 56b0038
OMR: 36bc5b0
OpenJDK11: 1f381a8 - openj9-openjdk-jdk11
linux_x86-64_cmprssptrs
```
Although it could be updated to protect the addition similar to how this deleted line worked.